src/ray/util/tests/spdlog_fd_sink_test.cc

```diff
@@ -0,0 +1,57 @@
+// Copyright 2025 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ray/util/spdlog_fd_sink.h"
+
+#include <gtest/gtest.h>
+
+namespace ray {
+
+namespace {
+
+#if defined(__APPLE__) || defined(__linux__)
+int GetStdoutHandle() { return STDOUT_FILENO; }
+#elif defined(_WIN32)
+HANDLE GetStdoutHandle() { return GetStdHandle(STD_OUTPUT_HANDLE); }
+#endif
```

Comment:
I think we already define this somewhere else?

Reply:
I'm working on a PR right now to have compatible function for file descriptor / handle on windows and unix platform, including: (1) file descriptor; (2) write/flush/read/close syscalls. We could migrate afterwards